client/reader/following/view-preference.ts

```diff
@@ -0,0 +1,24 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useDispatch, useSelector } from 'calypso/state';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getPreference } from 'calypso/state/preferences/selectors';
+
+export const READER_FOLLOWING_VIEW_PREFERENCE = 'reader_following_view';
+export type ReaderFollowingView = 'recent' | 'stream';
+export const DEFAULT_VIEW: ReaderFollowingView = 'stream';
+
+export const useFollowingView = () => {
+	const dispatch = useDispatch();
+	const currentView = useSelector(
+		( state ) => getPreference( state, READER_FOLLOWING_VIEW_PREFERENCE ) ?? DEFAULT_VIEW
+	);
+
+	const setView = ( view: ReaderFollowingView ) => {
+		dispatch( savePreference( READER_FOLLOWING_VIEW_PREFERENCE, view ) );
+		recordTracksEvent( 'calypso_reader_following_view_toggle', {
```

Comment:
The track event is not firing for me locally, and I'm also not seeing any error messages in the console. Is this behavior expected in local environment?

Reply:
No, you should see them fire locally. Are you following these instructions?  PCYsg-cae-p2

Reply:
Which instructions? I have looked at the Network Panels and no Tracks event was fired. It could be due to one of my extension or browser settings. If you're confident that it's working fine, then it's good to go.

Reply:
I'll share the setup steps in Slack.